### PR TITLE
Fixes for settings tab

### LINKED ISSUE #64: Obsolete "Screenshot background color"
As seen in the screenshot, seems the screenshot background color is obsolete in one of the settings?

screenshot background

### PR DESCRIPTION
**Pull request by james-tindal**
Fixes #64 and #65
image

### CODE REVIEW

**Comment by james-tindal** [comment]
Fixes #62 and #69

**Comment by james-tindal** [comment]
Fixed #66

**Comment by AdrianSimionov** [comment]
As @james-tindal used only one branch for multiple issues, this branch should not be squashed when merged to main.

File: package.json [hunk 1]
```diff
@@ -38,8 +38,10 @@
     "@codemirror/state": "^6.1.4",
     "@codemirror/view": "^6.6.0",
     "@types/d3": "^7.4.0",
+    "@types/d3-zoom": "^3.0.2",
     "d3": "^7.6.1",
     "d3-svg-to-png": "^0.2.1",
+    "d3-zoom": "^3.0.0",
```
Comment (MasterTuto) [inline package.json:44]: Did you use these dependency? What it the purpose of adding this?

**Review by MasterTuto — CHANGES_REQUESTED** [review]
Thanks for submitting your PR, we're very glad.

Do the changes requested so I can proceed with your PR.

File: package.json (hunk 1 above)
Comment (james-tindal) [reply package.json:44]: It's already used in mindmap-view.ts. It just wasn't in the dependencies.

File: src/main.ts [hunk 2]
```diff
@@ -14,17 +14,17 @@ const DEFAULT_SETTINGS = {
...
+  color1: "#cb4b16",
   color1Thickness: "8",
 
-  color2: "#2ab7ca",
+  color2: "#6c71c4",
   color2Thickness: "6",
 
-  color3: "#fe4a49",
+  color3: "#859900",
   color3Thickness: "4",
```
[the remark this answers is not in the document]
Comment (james-tindal) [reply src/main.ts]: I've changed thicknesses 1, 2 and 3

File: package.json (hunk 1 above)
Comment (james-tindal) [reply package.json]: I removed the dependency on d3-zoom but kept the types

**Review by MasterTuto — APPROVED** [review]

**Comment by MasterTuto** [comment]
Only the squashing is available, rebasing gives me conflicts that can only be fixed offline, but main branch is protected.

image

**Comment by AdrianSimionov** [comment]
Rebase and merge should be fine once conflicts are resolved.

**Comment by james-tindal** [comment]
Closing in favour of #90

**Pull request closed without merging**